deployment/grafana-dashboards/Hive-Vitals.json

```diff
@@ -361,6 +361,170 @@
       "title": "Postgres Overview",
       "type": "timeseries"
     },
+    {
+      "datasource": {
+        "type": "grafana-azure-monitor-datasource",
+        "uid": "u65mwBI4z"
```

Comment:
Isn't that id specific to an environment?

Reply:
fixed